controller/tenants_updates_blackbox_test.go

```diff
@@ -0,0 +1,341 @@
+package controller_test
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/jinzhu/gorm"
+	uuid "github.com/satori/go.uuid"
+
+	apptest "github.com/fabric8-services/admin-console/app/test"
+	"github.com/fabric8-services/admin-console/application"
+	"github.com/fabric8-services/admin-console/auditlog"
+	"github.com/fabric8-services/admin-console/configuration"
+	"github.com/fabric8-services/admin-console/controller"
+	testconfig "github.com/fabric8-services/admin-console/test/generated/controller"
+	"github.com/fabric8-services/fabric8-common/resource"
+	testauth "github.com/fabric8-services/fabric8-common/test/auth"
+	testsuite "github.com/fabric8-services/fabric8-common/test/suite"
+
+	"github.com/goadesign/goa"
+	goajwt "github.com/goadesign/goa/middleware/security/jwt"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	gock "gopkg.in/h2non/gock.v1"
+)
+
+func newTenantUpdatesController(config controller.TenantUpdatesControllerConfiguration, db application.DB) (*goa.Service, *controller.TenantUpdatesController) {
+	svc := goa.New("search")
+	ctrl := controller.NewTenantUpdatesController(svc,
+		config,
+		db,
+	)
+	return svc, ctrl
+}
+
+type TenantUpdatesControllerBlackboxTestSuite struct {
+	testsuite.DBTestSuite
+	app *application.GormApplication
+}
+
+func (s *TenantUpdatesControllerBlackboxTestSuite) SetupSuite() {
+	s.DBTestSuite.SetupSuite()
+	s.app = application.NewGormApplication(s.DB)
+}
+
+func TestTenantUpdatesController(t *testing.T) {
+	resource.Require(t, resource.Database)
+	config := configuration.New()
+	suite.Run(t, &TenantUpdatesControllerBlackboxTestSuite{DBTestSuite: testsuite.NewDBTestSuite(config)})
+}
+
+func (s *TenantUpdatesControllerBlackboxTestSuite) TestShowTenantUpdates() {
+	// given
+	config := testconfig.NewTenantUpdatesControllerConfigurationMock(s.T())
+	config.GetTenantServiceURLFunc = func() string {
+		return "https://test-tenant"
```

Comment:
the tenant host shouldn't contain `https://` scheme - it should be just `f8tenant`

Reply:
ok, let me change that

Reply:
done in 02cc202 (using `http://`, but that should work as well, I believe)

Reply:
my point was that the host doesn't contain any scheme at all - it is  just `f8tenant`, so the whole URL would be: `f8tenant/api/update`

Reply:
the config map has been updated to include the `http://` scheme, so we should be good with that now